src/EFCore.MySql.Json.Microsoft/Storage/ValueConversion/Internal/MySqlJsonMicrosoftPocoValueConverter.cs

```diff
@@ -9,17 +9,17 @@ namespace Pomelo.EntityFrameworkCore.MySql.Json.Microsoft.Storage.ValueConversio
 {
     public class MySqlJsonMicrosoftPocoValueConverter<T> : ValueConverter<T, string>
     {
-        public MySqlJsonMicrosoftPocoValueConverter()
+        public MySqlJsonMicrosoftPocoValueConverter(JsonSerializerOptions jsonSerializerOptions)
             : base(
-                v => ConvertToProviderCore(v),
-                v => ConvertFromProviderCore(v))
+                v => ConvertToProviderCore(v, jsonSerializerOptions),
+                v => ConvertFromProviderCore(v, jsonSerializerOptions))
         {
         }
 
-        private static string ConvertToProviderCore(T v)
-            => JsonSerializer.Serialize(v);
+        private static string ConvertToProviderCore(T v, JsonSerializerOptions jsonSerializerOptions)
+            => JsonSerializer.Serialize(v, jsonSerializerOptions);
 
-        private static T ConvertFromProviderCore(string v)
-            => JsonSerializer.Deserialize<T>(v);
+        private static T ConvertFromProviderCore(string v, JsonSerializerOptions jsonSerializerOptions)
+            => JsonSerializer.Deserialize<T>(v, jsonSerializerOptions);
```

Comment:
I am not sure at the moment, that we want to force the use of a `jsonSerializerOptions` parameter. We might want to make it optional, falling back on the default options.